doc(CU-8695gnmc0): use server side apply for workerpool controller (#745)
spacelift-io/user-documentation: docs/concepts/worker-pools/kubernetes-workers.md

    To install the worker pool controller along with its CRDs, run the following command:

    ```shell
    kubectl apply --server-side -f https://downloads.spacelift.io/kube-workerpool-controller/latest/manifests.yaml
    ```

    !!!warning

>         It is important to use the `--server-side` flag here. The reason is that our CRDs definitions are heavy and

Reviewer: ```suggestion
        It is important to use the `--server-side` flag here. The reason is that our CRD definitions
```

>         contains long description fields. This is causing [issues](https://github.com/kubernetes-sigs/kubebuilder/issues/3460)

Reviewer: ```suggestion
        contains long field descriptions. This causes the apply to fail, as detailed [in this kubebuilder issue](https://github.com/kubernetes-sigs/kubebuilder/issues/3460),
```
